File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = get_int("Enter the height of pyramid: \n");
height:
    if (height < 1 || height > 8)
    {
        printf("Error: Please enter a number between 1 and 8!\n");
        height = get_int("Enter another value: \n");
        goto height;
    }
    int goingdown = height;
    int space_counter = 0;
    for (int i = 0; i < height; i++)  // outter loop for the height
    {
        goingdown--;// 4
        for (int j = 0; j < goingdown; j++)  // going through spaces
        {
            printf(" ");
            space_counter++;
        }
        for (int g = goingdown; g < height; g++)  // going through hashtags - spaces
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int, get_string

while (True):
    number = get_int("Height:\n")
    if number in range(1, 9):
        space = number-1
        hashtag = 1
        for i in range(number):  # double for loop
            # print spaces
            separator = ' ' * space
            print(separator, end='')  # spacing
            space -= 1
            # print hashtags
            for j in range(hashtag):
                print("#", end='')
            # print spaces between pyramids
            print('  ', end='')
            # print other side of pyramid
            for j in range(hashtag):
                print("#", end='')
            hashtag += 1
            print()  # new row
        exit()